coala_quickstart/generation/Project.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from coala_utils.Question import ask_question
+from coala_utils.Extensions import exts
+
+
+def get_project_dir(printer):
+    """
+    Retrieves the project directory from the user.
+
+    :param printer:
+        A ``ConsolePrinter`` object.
+    :return:
+        An absolute path to the user's project directory.
+    """
+    project_dir = ""
+
+    while not os.path.isdir(project_dir):
+        project_dir = ask_question(
+            "What is your project directory?",
+            default=os.getcwd())
+        project_dir = os.path.abspath(os.path.expanduser(project_dir))
```

Comment:
wouldn't it be better to provide ask_question with a conversion method and let it take care of reasking the user if the conversion method raises an exception?